.docker/app/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM php:7.4-fpm-alpine
```

Using args maybe we could create an extension point in order to test multiple PHP versions later.

```suggestion
ARG PHP_VERSION=7.4-fpm-alpine

FROM php:$PHP_VERSION
```